### PR TITLE
[SYCL] Update rpc-server.cpp to include SYCL backend

### PR DESCRIPTION
**Pull request by nickp27**
PR to address inclusion of SYCL backend for RPC server

### CODE REVIEW

File: examples/rpc/rpc-server.cpp [hunk 1]
```diff
@@ -81,6 +85,14 @@ static ggml_backend_t create_backend() {
     }
 #endif
 
+#elif GGML_USE_SYCL
```
Comment (rgerganov) [inline examples/rpc/rpc-server.cpp:88]: this will give a compile error if you don't remove the `#endif` on line 86

Comment (nickp27) [reply examples/rpc/rpc-server.cpp:86]: Should be resolved now!

**Review by rgerganov — APPROVED** [review]

**Comment by slaren** [comment]
Does this really work? The SYCL backend has extras that are initialized during `init_tensor`.

**Comment by nickp27** [comment]
It's currently failing - see https://github.com/ggerganov/llama.cpp/issues/7718, although I suspect that issue is only one of a number possibly, having spent a few hours trying to get it working and seeing that similar issues have persisted with whisper.cpp for the last couple of months

**Pull request merged**